src/main/webapp/js/navigation-loader.js

```diff
@@ -55,6 +55,15 @@ function addPublicFeed() {
   navigationElement.appendChild(createListItem(createLink('/feed.html', 'Public Feed')));
 }
 
+function addMap() {
```

Comment:
For this function I think your method comment should only need to mention that the map will be accessed via a link